db/email_templates/welcome_to_new_user_added_by_affiliate.html.erb

```diff
@@ -738,7 +738,7 @@ The first site you have access to is:<br>
         <td class="mcnImageCardTopImageContent" align="center" valign="top" style="padding-top:9px; padding-right:18px; padding-bottom:18px; padding-left:18px;">
         
             
-            <a href="https://www.youtube.com/watch?v=TnlpuudK_WY" title="Getting Started With DigitalGov Search" class="" target="_blank">
```

Comment:
Sorry I missed this DigitalGov reference earlier!

Reply:
NP! It was the title of the video over on YouTube anyway.